jwst/resample/resample_spec_step.py

```diff
@@ -109,6 +112,14 @@ def process(self, input):
         result.meta.asn.table_name = input_models[0].meta.asn.table_name
         result.meta.asn.pool_name = input_models[0].meta.asn.pool_name
 
+        # populate the result wavelength attribute
+        if isinstance(result, MultiSlitModel):
+            for slit_idx, slit in enumerate(result.slits):
+                result_wl = result.slits[slit_idx].wavelength
+                if len(result_wl) == 0 or np.all(result_wl == 0.0):
+                    wl_array = get_wavelengths(result.slits[slit_idx])
```

Comment:
So this is getting wavelengths that match the resampled space of the images?

Reply:
The `get_wavelengths` function will either just return the existing 2-D wavelength array that's already present in the input model (`result.slits[]`) or recompute from the wcs object, if the wavelength array doesn't exist. For NIRSpec FS and MOS data, which have the wavecorr correction applied if there's a point source in the slit, the wavecorr result only exists in the 2-D wavelength array, not in the wcs object. So I'm curious what source of wavelengths ends up getting used here. Do the slits in the `results` object from resample_spec already have resampled wavelengths computed and stored in the `.wavelength` attribute or do the wavelengths need to be computed by the `get_wavelengths` function, or what?

Reply:
yea, good point. I removed the condition because the code has not computed the wavs attribute for the drizzled result. so now it will always be computed and populated for both Multislit and Slit models.

Reply:
will run a new reg test since this now works for NRS an MIRI data

Reply:
tests running at: https://plwishmaster.stsci.edu:8081/job/RT/job/JWST-Developers-Pull-Requests/1369/